.gitignore

```diff
@@ -36,3 +36,9 @@ phpunit.xml
 
 # sqlite test database
 phinx_testing.sqlite3
+
+# ignore dotenv .env file
+.env
```

Comment:
why is this needed? shouldnt this only be relevant for project's config files, not for inside vendor?

Reply:
The dotenv file (which is a config file) is typically placed in the root, and is ignored, because it will most likely be different in every environment an application is deployed to.

The .env.example is committed to give people a starting point. The naming of this file is also fairly standard across various project.

I'm not sure what you're asking about the vendor directory.

Reply:
vendor/cakephp/phinx/.gitignore => this file
What does that relate to .env files that are in root dir?

Reply:
Sounds good.